tests/signUp/signUp.spec.js

```diff
@@ -0,0 +1,286 @@
+        await inputName.fill('Іванна')
+        await inputName.blur()
+        const errorMessageLatinLettersName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageLatinLettersName).toContain("Name is invalid")
+
+        // Negative: check Special Symbols
+        await inputName.fill('!@#$%^&*()_')
+        await inputName.blur()
+        const errorMessageSpecialSymbolsName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageSpecialSymbolsName).toContain("Name is invalid")
+
+        // Positive: check trim BUG HERE
+        // await inputName.fill(' Ivanna ', 'For this input should be implemented trim')
+    })
+
+    test ('Sign Up with INVALID data for Last Name input', async ({page})=>{
+        const popup = page.locator('app-signup-modal')
+        const inputLastName = popup.locator('#signupLastName')
+
+        // Negative: check required Last Name
+        await inputLastName.click()
+        await inputLastName.blur()
+        const errorMessageRequiredinputLastName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageRequiredinputLastName).toContain("Last name required")
+
+        // Negative: check Min length Last Name
+        await inputLastName.fill('D')
+        await inputLastName.blur()
+        const errorMessageMinLengthInputLastName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageMinLengthInputLastName).toContain("Last name has to be from 2 to 20 characters long")
+
+        // Negative: check Max Length Last Name
+        await inputLastName.fill('IvannaDubenskaAqaEngineer')
+        await inputLastName.blur()
+        const errorMessageMaxLengthInputLastName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageMaxLengthInputLastName).toContain("Last name has to be from 2 to 20 characters long")
+
+        // Negative: check Spaces Last Name
+        await inputLastName.fill('Ivanna D')
+        await inputLastName.blur()
+        const errorMessageSpacesInputLastName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageSpacesInputLastName).toContain("Last name is invalid")
+
+        // Negative: check Latin letters Last Name
+        await inputLastName.fill('Дубенська')
+        await inputLastName.blur()
+        const errorMessageLatinLettersInputLastName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageLatinLettersInputLastName).toContain("Last name is invalid")
+
+        // Negative: check Special Symbols Last Name
+        await inputLastName.fill('!@#$%^&*()_')
+        await inputLastName.blur()
+        const errorMessageSpecialSymbolsInputLastName = await popup.locator(".invalid-feedback").textContent()
+        await expect(errorMessageSpecialSymbolsInputLastName).toContain("Last name is invalid")
```

Comment:
use test.step for each rule or write dedicated test for each rule

now it is hard to read